psasim/tools/psa_autogen.py

```diff
@@ -146,18 +149,18 @@
             bs = open(str("psa_ff_bootstrap_" + str(partition_name) + ".c"), "w")
             bs.write("#include <psasim/init.h>\n")
             bs.write("#include \"" + symbols[0] + "\"\n")
-            bs.write("#include <signal.h>\n")
+            bs.write("#include <signal.h>\n\n")
             bs.write(qcode)
+            bs.write(nsacl)
+            bs.write(policy)
+            bs.write(versions)
             bs.write("\n")
-            bs.write(nsacl + "\n")
-            bs.write(policy + "\n")
-            bs.write(versions + "\n")
             bs.write(handlercode)
             bs.write("\n")
             bs.write("int main() {\n")
             bs.write(sigcode)
-            bs.write("   __init_psasim(psa_queues, 32, ns_allowed, versions, strict_policy);\n")
-            bs.write("   " + entry_point + "();\nfor(;;);\n}\n")
+            bs.write("    __init_psasim(psa_queues, 32, ns_allowed, versions, strict_policy);\n")
```

Comment:
Apologies, that is correct. I though that was a log file. 

But that gives me another question, have we run the generated files againist  `code_style.py`?

Reply:
`code_style.py` only runs on files that are checked into git. We explicitly exclude generated files (which are only checked in on release branches) because those tend not to respect the code style due to having lines that are too long. (Also due to historical files that have the old style of whitespace inherited from PolarSSL, but that's something we could fix easily, we just haven't taken the time to do it.)